vmck/backends/submission.py

```diff
@@ -2,23 +2,21 @@
 
 
 def task(job, options):
-    return {
+    job_task = {
         'name': 'submission-handler',
         'driver': 'docker',
         'config': {
-            'image': f'vmck/vagrant-vmck:{options["env"]["vagrant_tag"]}',
+            'image': f'vmck/vagrant-vmck:{options["manager"]["vagrant_tag"]}',
             'command': '/src/submission/launch.sh',
         },
-        'env': {
-            'VMCK_ARCHIVE_URL': options['env']['archive'],
-            'VMCK_SCRIPT_URL': options['env']['script'],
-            'VMCK_ARTIFACT_URL': options['env']['artifact'],
-            'VMCK_URL': settings.VMCK_URL,
-            'VMCK_CALLBACK_URL': options['env']['callback'],
-            'VMCK_JOB_ID': str(job.id),
-        },
+        'env': {"VMCK_" + k.upper(): v for k, v in options['env'].items()},
```

Comment:
You might as well send them as `VMCK_ARCHIVE_URL` etc from acs-interface, it makes it easier to follow what's going on.